wallet/grpc/server/src/lib.rs

```diff
@@ -0,0 +1,123 @@
+    kaspawalletd_server::Kaspawalletd, BroadcastRequest, BroadcastResponse, BumpFeeRequest, BumpFeeResponse,
+    CreateUnsignedTransactionsRequest, CreateUnsignedTransactionsResponse, GetBalanceRequest, GetBalanceResponse,
+    GetExternalSpendableUtxOsRequest, GetExternalSpendableUtxOsResponse, GetVersionRequest, GetVersionResponse, NewAddressRequest,
+    NewAddressResponse, SendRequest, SendResponse, ShowAddressesRequest, ShowAddressesResponse, ShutdownRequest, ShutdownResponse,
+    SignRequest, SignResponse,
+};
+use tonic::{Request, Response, Status};
+
+#[derive(Debug, Default)]
+pub struct KaspaWalletService {
+    // Add your service state here
+}
+
+#[tonic::async_trait]
+impl Kaspawalletd for KaspaWalletService {
+    async fn get_balance(&self, _request: Request<GetBalanceRequest>) -> Result<Response<GetBalanceResponse>, Status> {
+        let response = GetBalanceResponse { available: 0, pending: 0, address_balances: vec![] };
+        Ok(Response::new(response))
+    }
+
+    async fn get_external_spendable_utx_os(
+        &self,
+        _request: Request<GetExternalSpendableUtxOsRequest>,
+    ) -> Result<Response<GetExternalSpendableUtxOsResponse>, Status> {
+        let response = GetExternalSpendableUtxOsResponse { entries: vec![] };
+        Ok(Response::new(response))
+    }
+
+    async fn create_unsigned_transactions(
+        &self,
+        _request: Request<CreateUnsignedTransactionsRequest>,
+    ) -> Result<Response<CreateUnsignedTransactionsResponse>, Status> {
+        let response = CreateUnsignedTransactionsResponse { unsigned_transactions: vec![] };
+        Ok(Response::new(response))
+    }
+
+    async fn show_addresses(&self, _request: Request<ShowAddressesRequest>) -> Result<Response<ShowAddressesResponse>, Status> {
+        let response = ShowAddressesResponse { address: vec![] };
+        Ok(Response::new(response))
+    }
+
+    async fn new_address(&self, _request: Request<NewAddressRequest>) -> Result<Response<NewAddressResponse>, Status> {
+        let response = NewAddressResponse { address: "".to_string() };
+        Ok(Response::new(response))
+    }
+
+    async fn shutdown(&self, _request: Request<ShutdownRequest>) -> Result<Response<ShutdownResponse>, Status> {
+        let response = ShutdownResponse {};
+        Ok(Response::new(response))
+    }
+
+    async fn broadcast(&self, _request: Request<BroadcastRequest>) -> Result<Response<BroadcastResponse>, Status> {
+        let response = BroadcastResponse { tx_i_ds: vec![] };
+        Ok(Response::new(response))
+    }
+
+    async fn broadcast_replacement(&self, _request: Request<BroadcastRequest>) -> Result<Response<BroadcastResponse>, Status> {
+        let response = BroadcastResponse { tx_i_ds: vec![] };
+        Ok(Response::new(response))
+    }
+
+    async fn send(&self, _request: Request<SendRequest>) -> Result<Response<SendResponse>, Status> {
+        let response = SendResponse { tx_i_ds: vec![], signed_transactions: vec![] };
```

Comment:
tx_i_ds is interesting. Might be worth renaming in the proto to `txIds, txId` so that the rust naming transform makes it `tx_ids, tx_id`

Reply:
renamed https://github.com/aspectron/rusty-kaspa/pull/144/commits/39b8738c410d029212b3c614f86ef541010044c4